.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,89 @@
+
+As you read and fulfill each of the following requirements below,
+put an "x" between the square brackets to mark each task as done, like so: [x]
+-->
+
+- [ ] I have read the **issue reporting section** in the [contributing guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md), so I know how to submit a good bug report with the required information
+- [ ] I have verified that the **issue is not fixed and is reproducible** in the **[latest version](https://www.qbittorrent.org/download.php)**
+- [ ] (optional but recommended) I have verified that the **issue is not fixed and is reproducible** in the **[latest CI build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci.yaml?query=branch%3Amaster+event%3Apush)**
+- [ ] I have **checked the [frequent/common issues list](https://github.com/qbittorrent/qBittorrent/projects/2)** and **searched** the issue tracker for similar bug reports (including closed ones) **to avoid posting a duplicate**
+- [ ] This report is **not a support request or question**, both of which are better suited for either the [discussions section](https://github.com/qbittorrent/qBittorrent/discussions), [forum](https://qbforums.shiki.hu/), or [subreddit](https://www.reddit.com/r/qBittorrent/). The [wiki](https://github.com/qbittorrent/qBittorrent/wiki) did not contain a suitable solution either
+- [ ] I have **pasted/attached the settings file and relevant log(s)** in the **Attachments** section at the bottom of the report. Mandatory: the settings file and at least the most recent log. See [this wiki page](https://github.com/qbittorrent/qBittorrent/wiki/Frequently-Asked-Questions#Where_does_qBittorrent_save_its_settings) if you're not sure where to find them.
+
+## Description
+
+<!--
+################################## IMPORTANT ##################################
+
+Delete each "(type here)" indicator and type your text in their place in the subsections below.
+You MUST fill in ALL subsections marked with "(type here)" with the appropriate information.
+
+Please make sure the description is worded well enough to be understood.
+Provide steps to reproduce the issue, any additional relevant information, suggested solution (if applicable) and as much context and examples as possible.
+For more information consult the Contributing Guidelines at https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md.
+
+Do not forget about the mandatory attachments!
+
+Use the Preview tab before posting to make sure your report looks like it is formatted properly.
+You don't need to delete these comments, they won't show up in the final post.
+-->
+
+### qBittorrent info and operating system(s)
+
+<!--
+IMPORTANT:
+if you did not get the qBittorrent installer from the links in the official website
+or if you did not install it from the PPA, please mention that after the version
+-->
+
+- qBittorrent version: (type here)
+- Operating system(s) where the issue occurs: (type here)
```

Comment:
After reading a few issues, these "qBittorrent version:" and "Operating system(s) where the issue occurs:" are disrupting the flow of reading. I can't easily filter them to get to the juicy part and furthermore i think they are unnecessary. A simple "(type here)" is sufficient IMO.
Same goes for the section below.